runtime/runtime/src/lib.rs

```diff
@@ -1245,9 +1308,60 @@ mod tests {
                 root,
                 &Some(validator_accounts_update),
                 &apply_state,
-                &[Receipt::new_refund(&account_id, small_refund)],
+                &[Receipt::new_refund(&alice_account(), small_refund)],
                 &[],
             )
             .unwrap();
     }
+
+    #[test]
+    fn test_apply_delayed_receipts() {
```

Comment:
Would be useful to have tests for corner cases, like: 
1) we have local and new receipts but there are so many delayed receipts that we only process those;
2) we run out of delayed receipts and continue processing receipts normally (test_apply_delayed_receipts stops right after the delayed receipts are all processed)

Reply:
added